e2e/ansible/roles/k8s-openebs-operator/tasks/main.yml

```diff
@@ -54,6 +54,13 @@
   with_items:
     - node-disk-manager-amd64
 
+- name: Change the openebs analytics value of openebs resources in operator YAML
+  replace:
+    path: "{{ ansible_env.HOME }}/{{ openebs_operator_alias }}"
```

Comment:
Remember some concern around using ansible_env.HOME in case of gke (or was it only for utils_path?)